src/platform_impl/ios/view.rs

```diff
@@ -222,12 +230,45 @@ declare_class!(
             };
 
             // Flip the velocity to match macOS.
```

Comment:
This is no longer done?

Reply:
Actually, after testing some more, I'm bringing it back along with some additional translation work to make iOS pinch/rotation events match macOS